tests/yam/agama/auto.pm

```diff
@@ -8,13 +8,15 @@ use base 'y2_installbase';
 use strict;
 use warnings;
 
-use testapi 'assert_screen';
+use testapi;
```

Comment:
We'd better to include functions which used, so recover it will be ok.

Reply:
yes, please, do not import the whole library, so we limit the number of function to use, better style.